sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -4448,7 +4448,7 @@ def hpat_pandas_series_count(self, level=None):
 
     Limitations
     -----------
-    - Parameter level is currently unsupported by Intel Scalable Dataframe Compiler
+    Parameter ``level`` is supported only with default value ``None``.
```

Comment:
Is it new correct way to define unsupported parameters?

Reply:
I'm following the point `shouldn't be phrase unsupported in SDC` in the checklist.

Reply:
Moreover we cannot say that a parameter is unsupported at all, at the least the default value is valid

Reply:
This point in checklist was about ``Unsupported by Intel SDC.`` which appears when there is no corresponding documentation in SDC for method from Pandas.

Reply:
Fixed